src/options/SMTConfig.cc

```diff
@@ -602,7 +603,7 @@ namespace opensmt {
   SMTConfig::parseCMDLine( int argc
```

Comment:
I think `parseCMDline`, `printHelp` (and `printVersion`) should be removed from this file and instead should be written from scratch in `bin/opensmt.cc`.